modules/programs/sheldon.nix

```diff
@@ -0,0 +1,82 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.sheldon;
+  tomlFormat = pkgs.formats.toml { };
+  sheldonCmd = "${config.home.profileDirectory}/bin/sheldon";
```

Comment:
What exactly is the difference between this and using `${cfg.package}/bin/sheldon`?